templates/.overcommit.yml

```diff
@@ -0,0 +1,8 @@
+CommitMsg:
+  ALL:
+    enabled: false
+
+PreCommit:
+  Rake:
+    enabled: true
+    command: ['bundle', 'exec', 'bin/rails']
```

Comment:
`bin/rake` ? Does `bin/rails` just alias to `bin/rake` in Rails5?

Reply:
Yup! More details here: https://github.com/rails/rails/issues/18878